src/UI/Component/Item/Factory.php

```diff
@@ -0,0 +1,78 @@
+<?php
+namespace ILIAS\UI\Component\Listing;
```

Comment:
The namespace MUST always conform with PSR-4, see https://github.com/ILIAS-eLearning/ILIAS/tree/trunk/src. So yes, my bad.